src/app/trainee/trainee-details/trainee-details.component.scss

```diff
@@ -0,0 +1,17 @@
+@import "~nhsuk-frontend/packages/core/settings/colours";
+.trainee-details {
+  max-height: 70vh;
+  overflow: auto;
+
+  &.large-display {
+    max-height: unset;
+  }
+}
+::ng-deep {
+  .mat-list-base .mat-list-item .mat-list-text .mat-caption {
```

Comment:
does this selector need to be this deep?

Reply:
yeah, material selector is this deep and as per css selector hierarchy overrides this (like the important we discussed). the ng-deep is also because material css targets its own [_ngcontent-ykr-c169] kinda attribute.

Reply:
okie